entities/params/postsignup.go

```diff
@@ -0,0 +1,11 @@
+package params
+
+// PostSignUp represents request body for POST /api/signup
+type PostSignUp struct {
+	Email         string `form:"email" validate:"required,email"`
+	Password      string `form:"password" validate:"required,min=8"`
+	TokenResponse string `form:"token_response" validate:"optional"`
+}
+
+func (p *PostSignUp) TrimSpaces() {
```

Comment:
It's usually okay to add a comment here inside the method explaining that we don't need to do any opertion, like:

```go
func (p *PostSignUp) TrimSpaces() {
// no-op
}
```